File: Frontend/projects/edu-sharing-api/src/lib/api/models/comment.ts
/* tslint:disable */
/* eslint-disable */
import { NodeRef } from './node-ref';
import { UserSimple } from './user-simple';
export interface Comment {
    comment?: string;
    created?: number;
    creator?: UserSimple;
    ref?: NodeRef;
    replyTo?: NodeRef;
}

File: Backend/services/core/src/main/java/org/edu_sharing/restservices/comment/v1/model/Comment.java
package org.edu_sharing.restservices.comment.v1.model;

import lombok.Data;
import org.edu_sharing.restservices.shared.NodeRef;
import org.edu_sharing.restservices.shared.UserSimple;

import com.fasterxml.jackson.annotation.JsonProperty;

@Data
public class Comment {
	private NodeRef ref;
	private NodeRef replyTo;
	private UserSimple creator;
	private long created;
	private String comment;
}
